service/src/main/kotlin/xyz/lebkuchenfm/external/storage/mongo/repositories/XSoundsMongoRepository.kt

```diff
@@ -65,12 +74,16 @@ data class XSoundEntity(
         null,
         sound.name,
         sound.url,
-        0,
+        sound.timesPlayed,
```

Comment:
I maintain the opinion that when we create new entity then timesPlayed should be `0` ;p

Reply:
First - I don't like that we have this as a constructor. We should have a `fromDomain` method (or something like that).
Second - @gorzalczany I don't agree. New entity should have `timesPlayed=0`, but it's not the responsibility of the repository to decide if the sound is new or not. That's done in the service. Here you don't even know that the sound is new. It might as well be imported from somewhere, or something like that. Service passes a domain object, repository saves it, does not think about where it comes from, or what it is exactly.

Reply:
This constructor/method should only be doing mapping from domain object to database entity.